doc/usr/expl/history.md

```diff
@@ -0,0 +1,41 @@
+# History
+
+This project was initially developed in 2016 at a French startup company named
+Stupeflix. It took inspiration from the scene graph paradigm of [OSG] for which
+many clumsy in-house patches were maintained, essentially for video playback
+support and lazy resources management.
+
+The video playback logic was the first foundation to be extracted from these
+patches, and landed in a dedicated module named [sxplayer] (standing for
+"Stupeflix Player"), a free (LGPL) video playback library on top of FFmpeg.
+
+While it helped with maintenance, most of the OSG features remained unneeded,
+the Python bindings were unmaintained and painful to work with, the stack of
```

Comment:
What do you think of:
```
the *custom* Python bindings were unmaintained, painful to work with and tied to a specific version of OSG, preventing us from using a more recent version. The stack of patches added on top of the project were also tied to the same specific version of OSG and were particularly challenging to rebase due to their complexity ?
```

Reply:
Sure, replaced. I just replaced the "were" into "was", and "their" into "its" (*the* stack).

Unrelated but I also added "game engine" after the first mention of OSG in the first paragraph